spec/controllers/aspect_memberships_controller_spec.rb

```diff
@@ -2,12 +2,12 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
-require 'spec_helper'
+require "spec_helper"
 
-describe AspectMembershipsController, :type => :controller do
+describe AspectMembershipsController, type: controller do
```

Comment:
Missing `:`. Have a look at our wiki on how to run the tests locally: https://wiki.diasporafoundation.org/Testing_workflow